library/std/src/sys/thread_local/statik.rs

```diff
@@ -14,12 +14,12 @@ pub macro thread_local_inner {
     (@key $t:ty, const $init:expr) => {{
         const __INIT: $t = $init;
 
+        // NOTE: this cannot import `LocalKey` or `Storage` with a `use` because that can shadow
+        // user provided type or type alias with a matching name. Please update the shadowing test
+        // in `tests/thread.rs` if these types are renamed.
```

Comment:
You can probably drop the first sentence, using `$crate` is pretty standard macro procedure.

Reply:
```suggestion
        // NOTE: Please update the shadowing test in `tests/thread.rs` if these types are renamed.
```